ProxyAgent/Properties/launchSettings.json

```diff
@@ -12,19 +12,22 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "REMOTE_ENDPOINT": "http://vm-snjz6.westus.cloudapp.azure.com",
-        "Key": "Value",
+        "REMOTE_ENDPOINT": "https://vm-123456.somecloud.cloudapp.azure.com",
```

Comment:
can you verify the old remote endpoint is gone?

Reply:
yep, verified already, it's gone